includes/RequestSSLViewer.php

```diff
@@ -391,7 +391,7 @@ public function getForm( int $requestID ): ?RequestSSLOOUIForm {
 		$htmlForm->suppressDefaultSubmit();
 		$htmlForm->setSubmitCallback(
 			function ( array $formData, HTMLForm $form ) {
-				return $this->submitForm( $formData, $form );
+				return $this->submitForm( $formData, $form, $this->context );
```

Comment:
No need to do this actually: submitForm is a member function, it can just $this->context, no need to pass as argument.